pkg/scheduler/apis/config/scheme/scheme_test.go

```diff
@@ -87,6 +87,15 @@ profiles:
   - name: VolumeBinding
     args:
       bindTimeoutSeconds: 300
+  - name: NodeAffinity
+    args:
+      addedAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+          - matchExpressions:
```

Comment:
Forgot one thing: is the merging semantics "smart"? For example:

- will "foo in [bar]" (user-specified) and "foo in [bar]" (added from pluginArgs) be merged into one entry, or two entries?
- will "foo in [bar]" (user-specified) and "foo in [baz]" (added from pluginArgs) be merged into "foo in [bar, baz]"?